src/OpenLoco/S5/S5.h

```diff
@@ -107,6 +123,93 @@ namespace OpenLoco::S5
 #pragma pack(pop)
     static_assert(sizeof(SaveDetails) == 0xC618);
 
+#pragma pack(push, 1)
+    struct Company
+    {
+        uint8_t pad_0000[0x8FA8];
```

Comment:
The existing company struct is already this size.

Reply:
We need separate structs for S5 so that we can change the in-game company struct size at a later date without affecting the load / save code. See how OpenRCT2 does this, `RCT1Ride`, `RCT2Ride`, `Ride`.

Reply:
It would also be nice if we could support different save formats as well, such as those created with the long station patch applied. This is way off, still, though; we'll need to implement quite a bit more before we can start thinking about raising station limits, I think.

Reply:
How did the patch achieve this?

Reply:
If I recall correctly, it repurposes the station memory to allow for only 128 stations instead of 256, but at double the length each.

Reply:
How would we detect whether it is a patched saved game or not?

Reply:
It might be hard to do automatically, yeah… @marijnvdwerf Do you remember whether the patch altered a version number in the SV5 or something?